onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -630,6 +640,17 @@ def _where_input_wrangler(
     TorchLibOpInfo("isnan", core_ops.aten_isnan),
     TorchLibOpInfo("isneginf", core_ops.aten_isneginf),
     TorchLibOpInfo("isposinf", core_ops.aten_isposinf),
+    TorchLibOpInfo(
+        "linalg.vector_norm",
+        linalg_ops.aten_linalg_vector_norm,
+        trace_only=True,
+        tolerance={torch.float16: (2e-3, 2e-3)},
+        input_wrangler=_linalg_vector_norm_input_wrangler,
+    ).skip(
+        matcher=lambda sample: sample.kwargs.get("ord") == 6
+        and sample.input.dtype == torch.float16,
+        reason="ORT return wrong value for float16 with ord=6 (expected=Inf, actual=9.48).",
```

Comment:
Looks like the number we get is actually more accurate. Is that true?

Reply:
We can make it an xfail here potentially, so we know it when things get fixed

Reply:
If ORT is more accurate, we can change the word "wrong value".

Reply:
Done